go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/kubevirt/cluster-network-addons-operator
 
-go 1.22.0
-
-toolchain go1.22.2
+go 1.23.0
```

Comment:
interesting that running the bumper.sh removed it (well it just does make vendor and such unless something delicate happens there)
`./hack/bumper.sh CVE-2025-22869 golang.org/x/crypto@v0.36.0 main`